apps/backend/src/modules/auth/auth.service.ts

```diff
@@ -11,6 +14,29 @@ class AuthService {
 		this.userService = userService;
 	}
 
+	public async signIn(
+		userRequestDto: UserSignInRequestDto,
+	): Promise<UserSignInResponseDto> {
+		const { email, password } = userRequestDto;
+		const user = await this.userService.find({ email });
+
+		if (!user) {
+			throw new HTTPError({
```

Comment:
You should create a new exception. Each module should have its own exception

Reply:
I've created an AuthError class. Should it remain classified as a COMMON error,
like response for client:
{
    "errorType": "COMMON",
    "message": "Incorrect credentials."
}

 or should I handle it specifically in the initErrorHandler method of the BaseServerApplication class?

Reply:
What is errorType? A new class should extend HTTPError class

Reply:
Yes its extends from HTTPError and we have "errorType": "COMMON" for client.
Do I need add  errorType : "AUTH" and handle this in ErrorHandler ?

Reply:
You can keep COMMON